lib/sassc/native.rb

```diff
@@ -42,7 +42,7 @@ module Native
 
     # Remove the redundant "sass_" from the beginning of every method name
     def self.attach_function(*args)
-      super if args.size != 3
+      return super if args.size != 3
```

Comment:
If `args.size != 3`, the super method is called twice. It is meaningless, so it should return after super is called.

This change suppresses the following warnings.


```console
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old version
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old version
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old _make_data_context
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old _make_data_context
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old _context_get_included_files
/home/pocke/.rbenv/versions/trunk/lib/ruby/gems/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:275: warning: method redefined; discarding old _context_get_included_files
```